demos/continuous_batching/.gitignore

```diff
@@ -0,0 +1,3 @@
+vllm
```

Comment:
why this is included here?

Reply:
We rarely use gitignores in this repository. I think it is useful to keep directories and files in gitignores so that when you run the demo, the files generated in the demo (model files, venvs, configs, graphs) are ignored by git. @dtrawins